projects/angular/src/modal/modal-configuration.service.ts

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright (c) 2016-2024 Broadcom. All Rights Reserved.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { Injectable } from '@angular/core';
+
+@Injectable()
+export class ClrModalConfigurationService {
+  fadeMove = 'fadeDown';
```

Comment:
As we talked we can use better naming here but it can be done on later improvements too. Enum for the values would be good too.

Reply:
Yeah, would be nice but I don't want to overload this already biggish review.
We could add it separately.